packages/vue-labs/src/components/FTable/FTable.vue

```diff
@@ -68,6 +68,14 @@ const rowsFromPaginator = ref(metaRows.value);
 const viewRows = computed((): Array<MetaRow<T>> => {
     return paginerated ? (rowsFromPaginator.value as Array<MetaRow<T>>) : metaRows.value;
 });
+const isEmpty = computed((): boolean => {
+    return !viewRows.value.length;
```

Comment:
Nitpick: jag tycker syftet blir tydligare om man jämför med `=== 0` om än det är korrekt annars. Lite som att läsa engelska istället "om längden är 0".

```suggestion
    return !viewRows.value.length;
```